frontend/app/src/components/settings/data-security/data-management/PurgeData.vue

```diff
@@ -129,17 +131,17 @@ const { status, pending, showConfirmation } = useCacheClear<Purgeable>(
           item-value="id"
           :disabled="pending"
         />
-        <LocationSelector
-          v-if="source === Purgeable.EVM_TRANSACTIONS"
-          v-model="evmChainToClear"
+        <ChainSelect
+          v-if="source === Purgeable.TRANSACTIONS"
+          :model-value.sync="chainToClear"
```

Comment:
I was wondering if this will be automigrated in to v-model in vue 3. I will have to check.